pyanaconda/modules/payload/live/live_os.py

```diff
@@ -100,3 +122,55 @@ def teardown_installation_source_with_task(self):
         task = TeardownInstallationSourceTask(INSTALL_TREE)
 
         return self.publish_task(LIVE_OS_HANDLER.namespace, task)
+
+    def pre_install_with_task(self):
```

Comment:
I suggest to support multiple tasks in the payload installation methods and rename the DBus API according to that. You will need multiple tasks at some point anyway and it will be more difficult to do that later.

Reply:
This will be cleaner when done in a separate PR with the creation of the shared API.